trollmoves/movers.py

```diff
@@ -296,10 +298,10 @@ class ScpMover(Mover):
 
     def open_connection(self):
         """Open a connection."""
-        from paramiko import SSHClient, SSHException
 
         retries = 3
         ssh_key_filename = self.attrs.get("ssh_key_filename", None)
+        timeout = self.attrs.get("timeout", None)
```

Comment:
I think the config variable name should be "ssh_connection_timeout", since it only applies to the ssh connection, right?